README.md

```diff
@@ -1,26 +1,63 @@
-# streetscapes
-This repository contains information and code for retrieving and using data from the [global-streetscapes](https://github.com/ualsg/global-streetscapes/tree/main) dataset. 
+# `streetscapes`
+This repository contains information and code for retrieving and using data from the [global-streetscapes](https://github.com/ualsg/global-streetscapes/tree/main) dataset.
 
-### Converting global-streetscapes to parquet
+# Installation
+Clone the repository, enter the root directory and use pip to install the `streetscapes` package:
 
-`convert_csv_to_parquet.py` contains the code for reproducing `streetscapes-data.parquet` dataset. 
-For input you will have to download the original csv files from [hugging face](https://huggingface.co/datasets/NUS-UAL/global-streetscapes/tree/main/data). The code looks for these csv files in `../streetscapes-data`, which you may need to create yourself or change the path to the location of the files. 
-To limit size, the dataset contains the following data:
+```shell
+$> git clone git@github.com:Urban-M4/streetscapes.git
+<...>
+$> cd streetscapes
+$> pip install -e .
+```
 
-    - `contextual.csv`
-    - `metadata_common_attributes.csv`
-    - `segmentation.csv`
-    - `simplemaps.csv`
+### Dependencies
+There are a lot more dependencies in `pyproject.toml` than strictly necessary to run the examples in this repository. They are necessary for running (at least part of) the code in the original `global-streetscapes` repository, specifically the training pipeline (WIP).
 
-It is possible to combine more CSV files if needed. 
+Streetscapes uses a [custom version](https://github.com/Urban-M4/mapillary-python-sdk) of the [Mapillary Python SDK](https://github.com/mapillary/mapillary-python-sdk) which fixes some dependency issues.
 
-### Analysing the data
+## Environment variables
```

Comment:
I think it would be nice if the notebooks that don't require mapillary can still be run without this throwing an error I think.

Reply:
Indeed, I'll make those optional.